lhotse/cut/__init__.py

```diff
@@ -7,28 +7,32 @@
 
 """
```

Comment:
This docstring should be moved to the top of the file (I think then it becomes a module-level docstring). Also--great visualisation!